docs/user-docs/latest/plugins/installing-plugins-via-marketplace.md

```diff
@@ -4,37 +4,41 @@ description: Instructions to install new plugins in a Strapi application via the
-:::
-
-The Marketplace is a section of the admin panel that lists all plugins that can be installed in a Strapi application. The Marketplace is accessible from ![Marketplace icon](../assets/icons/marketplace.svg) _Marketplace_ in the main navigation of the admin panel.
+The Marketplace contains optional plugins built to customize the core Strapi installation. The Marketplace is located in the admin panel, indicated by the ![Marketplace icon](../assets/icons/marketplace.svg) _Marketplace_. Marketplace plugins are installed using the CLI (see [Developer Documentation](/developer-docs/latest/developer-resources/cli/CLI.md#strapi-install)).
 
-<!--
+::: tip
+The Marketplace displays only v4 plugins. Both v3 and v4 plugins are discoverable in the [Strapi Market](https://market.strapi.io/). v3 and v4 plugins are not cross-compatible.
 
-::: note
-Plugins can also be installed via the Command Line Interface (see [Developer Documentation](/developer-docs/latest/developer-resources/cli/CLI.md#strapi-install)).
 :::
 
-![Interface of the Marketplace](../assets/plugins/marketplace.png)
+## The Marketplace
 
-The Marketplace displays each available plugin in a box, which contains:
-- the name of the plugin,
-- the description of the plugin,
-- indications on the current status of the plugin:
-   - "Compatible with your app": indicates that the plugin is not installed yet but can be installed on your Strapi application
-   - "Already installed": indicates that the plugin is already installed and available in your Strapi application
+![The Marketplace interface](../assets/plugins/marketplace-v4.png)
 
 ::: tip
-Click on the link icon ![External link icon](../assets/icons/external_link.svg) next to the name of a plugin to be redirected to the plugin package in the Strapi GitHub repository.
+The search bar displays find as you type results that search the plugin name and description. 
 :::
 
+Plugins are displayed on individual cards containing:
+
+- the plugin name
+- the plugin description
+- a _Learn more_ link for additional information, including detailed implementation instructions
+- a button to copy the install command to the local clipboard
+- a badge indicating if the plugin is _made by Strapi_ ![made by Strapi icon](../assets/icons/official-market.svg) or _verified by Strapi_ ![verified by Strapi icon](../assets/icons/verified-marketplace.svg)
+
+## Installing Marketplace plugins
+
 To install a new plugin via the Marketplace:
 
 1. Go to the ![Marketplace icon](../assets/icons/marketplace.svg) Marketplace.
-2. Among the available plugin, choose the one you wish to install.
-3. Click on the **Install** button in the chosen plugin's box.
+2. Choose an available plugin and click on the _Copy install command_ button.
+3. Switch to your CLI application and navigate to the Strapi application directory.
```

Comment:
I think "terminal" is best in this case. Used with a lower case, it could refer to any terminal-like application on any operating system (Windows, Mac, Linux).